run-nitro-enclaves-acm-tests

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+EXIT_FAIL=1
+
+# Binaries which should exist under $PATH after package installation
+binaries="nitro-cli \
+	vsock-proxy \
+	p11ne-agent \
+	p11ne-client"
+
+# Files brought in by the RPM(s), specified by full path
+files="/lib/systemd/system/nitro-enclaves-allocator.service \
+	/lib/systemd/system/nitro-enclaves-vsock-proxy.service \
+	/lib/systemd/system/nitro-enclaves-acm.service \
+	/usr/share/nitro_enclaves/p11ne/p11ne.eif \
+	/usr/share/nitro_enclaves/p11ne/image-measurements.json \
+	/usr/share/nitro_enclaves/p11ne/acm.example.yaml"
+
+
+function check_binaries() {
+	binaries="$1"
+
+	echo -n "Checking binaries exist in PATH .........."
+
+	for binary in $binaries; do
+		x="$(which $binary 2>&1)"
+		echo $x | grep "no $binary in" 2>&1 >/dev/null && echo " [FAIL]" && exit $EXIT_FAIL
```

Comment:
`"$binary"` and `> /dev/null 2>&1` instead

Reply:
Done in the latest revision.